### ISSUE TITLE
logs of pretrained models

### ISSUE THREAD

**Issue by faroit** [ws-choi/Conditioned-Source-Separation-LaSAFT#26]
@ws-choi would you mind sharing some loss curves for the pre-trained models or at least the final train and validation loss?

**Comment by ws-choi** [comment]
Hi @faroit,
Here is the [wandb report](https://wandb.ai/wschoi/lasaft/reports/LaSAFT-larger-model-training-eval-example--Vmlldzo1NDczMTM) that I have created for the large model with checkpoint ```2021```.

I think I have deleted the other sessions due to the storage limitation of [wandb](https://www.wandb.com/). If I found them I will share them too.

**Comment by faroit** [comment]
@ws-choi thats super helpful, thanks.

Maybe add the main loss curves as png to the repo so they will be archived here...

**Comment by ws-choi** [comment]
Thank you for your advice. 5f739650ed524acc1eff13a01a62276619008889
